lib/tapioca/dsl/compilers/active_record_columns.rb

```diff
@@ -264,12 +264,20 @@ def add_methods_for_attribute(klass, attribute_name, column_name = attribute_nam
             "saved_change_to_#{attribute_name}?",
             methods_to_add,
             return_type: "T::Boolean",
+            parameters: [
+              create_kw_opt_param("from", type: setter_type, default: "T.unsafe(nil)"),
+              create_kw_opt_param("to", type: setter_type, default: "T.unsafe(nil)"),
```

Comment:
I don't think there is any guarantee that `from` or `to` will have the `setter_type`. As a matter of fact, even [their default value](https://github.com/rails/rails/blob/main/activemodel/lib/active_model/attribute_mutation_tracker.rb#L44-L48) is a [sentinel object](https://github.com/rails/rails/blob/main/activemodel/lib/active_model/attribute_mutation_tracker.rb#L8), and they are also [processed through `type_cast`](https://github.com/rails/rails/blob/main/activemodel/lib/active_model/attribute_mutation_tracker.rb#L46), so I think they can be passed as any arbitrary type.

```suggestion
              create_kw_opt_param("from", type: "T.untyped", default: "T.unsafe(nil)"),
              create_kw_opt_param("to", type: "T.untyped", default: "T.unsafe(nil)"),
```